client/.env.sample

```
@@ -1 +1 @@
-REACT_APP_SENTRY_DSN=[leave empty on development]
+REACT_APP_SENTRY_DSN_FRONTEND=[leave empty on development]
```

```suggestion
REACT_APP_SENTRY_DSN=[leave empty on development]
```
Ignore me, I forgot these are kept in different .env files 🙄  it's been a long week for me, @arku don't judge me 😂